pageserver/src/tenant/timeline.rs

```diff
@@ -129,7 +129,7 @@ pub struct Timeline {
     ancestor_lsn: Lsn,
 
     // Metrics
-    metrics: TimelineMetrics,
+    pub metrics: TimelineMetrics,
```

Comment:
```suggestion
    pub(super) metrics: TimelineMetrics,
```

Using field access from the `gc_iteration_internal` requires the `pub(super)` on the timeline, but no `pub` usage from elsewhere.

Reply:
pub(super)